concepts/05 Widgets/FileManager/10 Bind to File Systems/10 Object File System.md

```diff
@@ -78,6 +78,194 @@ In the example below, the **FileManager** widget displays hierarchical data stor
     //...
     ];
 
+##### Angular
```

Comment:
Does this example differ from the one we give in the "Bind the File Manager Widget to a File System" article?

Reply:
Yes, with a small difference: in the current article the example illustrates binding to a data source with non conventional field names.
In the 'Getting Started' section we intentionally use the Object file provider as an example but in its simple version. See a note in the article (Getting Started).

Reply:
The difference in these articles is correct.

Reply:
I decided to leave two examples (simple and advanced binding) in the 'Object File System' topic as if a user enters documentation in Binding section (passing the Getting Started section), the user will see only one way of binding.